upstairs/src/lib.rs

```diff
@@ -1935,7 +2042,13 @@ impl GuestWork {
      * will be handled if all the downstairs have returned error.
      */
     #[instrument]
-    fn ds_complete(&mut self, gw_id: u64, ds_id: u64, data: Option<Bytes>) {
+    fn ds_complete(
```

Comment:
The TODO just above this can be removed now, as these changes should address it.

Reply:
removed in 8ce3a2a